core/src/main/resources/jenkins/monitor/JavaVersionRecommendationAdminMonitor/message.properties

```diff
@@ -1 +1,2 @@
-Recommended_Java_Version=Java 11 is the recommended version to run Jenkins on; please consider upgrading.
+Recommended_Java_Version=You are running Jenkins on Java {0}, support for which will end on or after June 14, 2022. \
```

Comment:
this date seems a bit far away? shouldn't it be after the next LTS baseline (9th March).

not 100% sure how to express but my thoughts would be that line will continue under Java 8 but the next weekly would drop support?

Reply:
> this date seems a bit far away? shouldn't it be after the next LTS baseline (9th March).

Please refer to the PR description:

> The date has not yet been finalized, hence this PR being in draft state. For the sake of example, I have chosen [Flag Day](https://en.wikipedia.org/wiki/Flag_day_(computing)) 2022. Once stakeholders like @MarkEWaite have made the decision, this PR can be updated with the final date.

> not 100% sure how to express but my thoughts would be that line will continue under Java 8 but the next weekly would drop support?

Different branches can have different dates.